lib/web_ui/dev/canvaskit_roller.dart

```diff
@@ -85,8 +85,6 @@ data:
 const List<String> _canvaskitAssets = <String>[
   'canvaskit.js',
   'canvaskit.wasm',
-  'profiling/canvaskit.js',
-  'profiling/canvaskit.wasm',
```

Comment:
My recommendation is to keep the profile build but remove all the runtime switcheroos in the engine. This way people can still profile their apps by pointing to the profile build using the `canvasKitBaseUrl` configuration parameter.

Reply:
Done